src/compile/unit.ts

```diff
@@ -320,51 +315,43 @@ export class UnitModel extends Model {
     return UNIT_CHANNELS;
   }
 
-  protected mapping() {
-    return this.encoding();
-  }
-
-  public stack(): StackProperties {
-    return this._stack;
+  protected getMapping() {
```

Comment:
Okay. I can rename them back. The problem was that I had not renamed scale to scales yet and so the I had a name conflict.